docarray/utils/find.py

```diff
@@ -114,7 +124,7 @@ def find_batched(
     limit: int = 10,
     device: Optional[str] = None,
     descending: Optional[bool] = None,
-) -> List[FindResult]:
+) -> FindResultBatched:
```

Comment:
You need to adapt the implementation to actually return a `FindResultBatdched` :)

Reply:
@JohannesMessner how would I handle `scores` here?

Reply:
I am not sure if this is the correct way to go about it but I used `parse_obj_as()` passing the embedding type so that scores are then parse as the correct tensor. Was this the correct way to go about this?